### DISCUSSION TITLE
Copilot chat not showing

### DISCUSSION THREAD

**Discussion by amalv** [community/community#50851] [Copilot Conversations] [answered]
I use Ubuntu 21.10 and today I've installed the latest version of Github Copilot extension after signing up. I use the latest version of Visual Studio Code (I've updated today just in case).

Copilot works inside the code, however the icon in the left tool bar does not show, so it does not allow me to chat with it.

I've tried several solutions like using the previous version of the Copilot extension and  using the nightly extension, but none of them worked.

I've also setup in settings the following options:

```
"github.copilot.enable": {
    "*": true,
    "typescript": true,
    "typescriptreact": true,     
}
```

Another thing a little off topic is that not all shortcuts work, like the following ones:

```
Show next inline suggestion | Alt+] | editor.action.inlineSuggest.showNext
Show previous inline suggestion | Alt+[ | editor.action.inlineSuggest.showPrevious
Trigger inline suggestion | Alt+\ | editor.action.inlineSuggest.trigger

```

I'm using VSC 1.76.2 and Copilot v1.78.9758

Any suggestions?

**Comment by ltalirz** [comment]
Same here - https://github.com/github-copilot/chat_waitlist_signup/join says "you're in" but "Chat" icon in toolbar does not show

**Reply by orikahmadov** [reply]
I have the same issue, probably we are all in waiting list and we  will be notified once the chat feature is enabled for us. But what i do not understand is, i am already paying monthly for GitHub Copilot why not just enabling the chat for every customer....

**Reply by veonua** [reply]
it works in the VisualStudio but no icon in the VSCode (for the same account)

**Reply by ArnaudC** [reply]
Is it possible to enable it with a VPN ?

**Reply by Areshra** [reply]
found the solution.

go to copilot settings on github website ( https://github.com/settings/copilot ) 
if you purchased copilot for business go to (https://github.com/settings/organizations) find the organization with copilot and click settings.
find copilot on the left and click the dropdown option policies and features. (https://github.com/organizations/[your_organization]/settings/copilot/policies).

check if all options are active.

thats all :)

**Reply by itLeeyw** [reply]
Thanks

**Reply by sdharri4** [reply]
Most of these answers were way to complicated for me I just simply closed the chat and couldnt find it so I click the burger stack/view and chat you can also use cntl + alt + i . Im sure im not the only one that only needed this answer

**Comment by sak797** [comment]
Using the nightly version with copilot labs on vs code works. doesn't work with normal extension

**Reply by phunguyen-mck** [reply]
does it show copilot chat extension icon on the left bar? @sak797  mine is not showing even with nightly version

**Reply by orikahmadov** [reply]
It is useless feature. We are looking for real chat feature that was demonstrated with the latest GitHub Copilot X

**Reply by sak797** [reply]
Sorry for the confusion. I think chat is enabled only for certain people at the moment.

**Reply by humbitious** [reply]
I'm not seeing the chat icon on the left -- in spite of seemingly being accepted into copilot (really hard to tell from github admin panel) -- even though I'm using VS Code Insiders and Nightly, etc. Been like this for weeks.

**Comment by nagibov** [comment]
I have a similar problem with Windows 11 using the latest version of VS and the Copilot extension. 
When I came inside my code editor and right-click to get the context menu I do not see the GitHub Copilot item on top. 
Also, the shortcut Ctr Shit A does not work. 

This is what GitHub Copilot  gives as output on start: 
GitHub Copilot 1.78.0.0 (v1.78.0.0@399602824)
GitHub Copilot Agent v1.78.0
[DEBUG] [agent] [2023-04-06T18:44:57.297Z] Agent service starting
[DEBUG] [agent] [2023-04-06T18:44:57.439Z] Telemetry initialized

GitHub Copilot 1.78.0.0 (v1.78.0.0@399602824)
GitHub Copilot Agent v1.78.0
[DEBUG] [agent] [2023-04-06T18:44:57.297Z] Agent service starting
[DEBUG] [agent] [2023-04-06T18:44:57.439Z] Telemetry initialized

**Comment by orikahmadov** [comment]
Can anyone from GitHub support please address above given issues? I am also not able to access the chat feature although I am subscriber to GitHub Copilot

**Comment by the-snowwhite** [comment]
I now have 24 days left of my free trial. 
And canceled my paid subscription due to the chat icon missing in ubuntu (KDE Neon 22.04) issue.
` You cancelled your free trial. Your access to GitHub Copilot expires in 24 days.`
I have tried both vscode, vscode insiders, github copilot and the nightly version.
Also downgrading the plugin versions to versions suggested elsewhere to work.
My current versions are:
vcode insiders: 

    Version: 1.78.0-insider
    Commit: e5512217bcfb2b644fe7ad6c465fe0b45d23265b
    Date: 2023-04-12T05:20:39.394Z
    Electron: 22.4.4
    Chromium: 108.0.5359.215
    Node.js: 16.17.1
    V8: 10.8.168.25-electron.0
    OS: Linux x64 5.19.0-32-generic
    Sandboxed: Yes

Github copilot nightly:

    v1.82.16

Could someone confirm If this issue is due to being on the waiting list, and still waiting for some kind of activation ?
If so can I expect the chat icon to be activated before my free trial runs out  (in 24 days) ?

**Reply by the-snowwhite** [reply]
Well I guess the answers are in this thread:
https://github.com/orgs/community/discussions/23504
I if replace GitHub Copilot with GitHub Copilot Chat ?

**Comment by amalv** [comment]
I've signed up through this url when copilot x was announced: https://github.com/features/preview/copilot-x

After I signed up through the wait list link, I had access to Github Copilot inside the code in VSC, but not the Github Copilot Chat in the left toolbar, so is the assumption that there is another different wait list for the chat that you have to sign up separately?

In the FAQ it says:

At this time, the technical preview is for VS or VSCode users. An active subscription to GitHub Copilot for Business or GitHub Copilot for Individuals is required for access to GitHub Copilot chat.

So it seems that if you sign up to the technical preview you would have access also to Github Copilot chat?

I feel like I'm missing something here, but maybe I'm just on a waiting list for the chat and it's simply a question of waiting.

In my personal account github settings it shows Github Copilot is active:

![test](https://user-images.githubusercontent.com/1252707/231856161-2c047261-f687-40a2-9acb-07c864644601.png)

And the charge for Github Copilot is also correctly shown under Billing and Plans > Plans and usage

**Comment by zvalcav** [comment]
Hi, I don't see chat feature neither. I've even tried to install VSCode on Windows without any change. According to this [site](https://visualstudiomagazine.com/articles/2023/03/24/visual-studio-chat.aspx), we have to join wait list for chat feature. I'm not sure it is actual, but I was able to join wait list for Visual Studio Code. Link is below Copilot chat screen.

**Comment by wellwind** [comment]
Copilot chat should be joined [waitlist](https://github.com/github-copilot/chat_waitlist_signup) first.

After the team lets you in, you will receive an email including instructions on how to install Copilot chat.

If you see the following message on the waitlist page, go check your email.

<img width="671" alt="image" src="https://user-images.githubusercontent.com/8186117/236613143-d5631f0e-ebad-4b3f-93dd-2531df0585aa.png">

**Reply by vaibhav3027** [reply]
I am already seeing this but, even after following the instructions mailed to me chat windows is not visible

**Reply by alexandrsashin** [reply]
You **need to use VS Code - Insiders** (not VS Code) that you downloaded on the first step of GitHub instruction.

**Reply by bamboodigital-dev** [reply]
I'm having the same issue, what do you mean vs code - insiders?

**Reply by alexandrsashin** [reply]
It's different version of VS Code (as different application). The key difference - using **VS Code - Insiders** you can use new features without new releases waiting. You can read more about it here: https://code.visualstudio.com/blogs/2016/05/23/evolution-of-insiders
![image](https://github.com/community/community/assets/16785779/0dace2fc-e882-4085-9fe7-9fcea6182b48)

**Reply by alexandrsashin** [reply]
> Previously, you had to use VS Code Insiders to use Copilot Chat. As of VS Code 1.79, you can use Copilot Chat in stable VS Code as well. You will still have to install the [GitHub Copilot Nightly](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-nightly) extension.

https://code.visualstudio.com/updates/v1_79#_github-copilot

**Reply by janupan** [reply]
@alexandrsashin I already install the vs code - insiders but the copilot chat still does not show. Can anyone explain what happened?
<img width="1440" alt="Screen Shot 2023-06-22 at 16 05 21" src="https://github.com/community/community/assets/18500720/e605bf2b-cb41-47c5-9be2-4fd1e137254b">

**Reply by alexandrsashin** [reply]
@janupan
Actual instruction:
1. Send an application to https://github.com/github-copilot/chat_waitlist_signup/join (not all users can use Copilot chat).
2. Get a message from Github with topic "You're now in the GitHub Copilot chat private beta!"
3. If you use VS Code you can:
- install [GitHub Copilot Nightly](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-nightly) but check that VS Code has v1.79+
- install **VS Code - Insiders** and [GitHub Copilot Nightly](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-nightly) extension
4. Log in VS Code through your GitHub account.
5. Chat icon will appear in VS Code sidebar.

**Comment by igorkasyanchuk** [comment]
Hello, I have an issue - the icon to start chat is not available.

Today received email: 
"Welcome to the Copilot chat private beta! You're off the waitlist and can start using Copilot chat"

I followed the instructions like installing the extension "Github Copilot Nightly". 
Please advise what I can do

**Reply by wellwind** [reply]
Have you installed VSCode Insider? It works only in Insider version now

**Reply by Zombieswilleatu** [reply]
the email i got says it works in visual studio too but i cant make it work either.

**Reply by Zombieswilleatu** [reply]
i got it to work i had to reauthorize my github credentials and it works now (click on your name in vs)

**Reply by igorkasyanchuk** [reply]
thanks! now works, I was confused with this "insiders" name. All works in the Insiders version.
But I have a question, do I need to install all extensions again here? can I load my profile and they will be installed?

**Reply by For-Chance** [reply]
Login in your Github accout and turn on setting sync.

**Reply by JeyFisher** [reply]
Thanks for noticing that it should be VS Code **Insiders** version, it's helped. I tried to run it in the standard version.

**Comment by jeannen** [comment]
Hey!
I just received the notification this morning, I installed VSCode Insider + Copilot nightly + Copilot chat but the chat don't appear on the side (I'm logged to my Github account that was approved for the BETA).

Is there something I'm missing?

**Reply by philippjbauer** [reply]
I installed the same as you a few days ago and initially it was working. Now I experience the same problem as you do. Copilot and inline chat works, the sidebar chat is empty though.

**Reply by pkalikman** [reply]
Ditto.

**Reply by gamousquesORT** [reply]
Same here, yesterday on Win computer worked fine, now macOs don't

**Reply by bamboodigital-dev** [reply]
its vs code insder + copilot main not nightly + copilot chat

**Comment by jenswilms** [comment]
Double-check the following
* Did you receive an email from GitHub with the title: "You're now in the GitHub Copilot chat private beta!"
* Did you install VS Code Insiders? 
* Do you have the Nightly extension?
* What did it for me (thanks @Zombieswilleatu): login again with your GitHub account by clicking on the profile icon in VSCode

**Reply by soupCodez** [reply]
still doesn't work for me...

**Reply by pkalikman** [reply]
Ditto.

**Reply by For-Chance** [reply]
All this I have done. And I try this chat once successfully. But after that, the chat icon disappeared.

**Reply by nilshulth** [reply]
Ditto

**Reply by sipuha** [reply]
the same here. done all of this but still no chat window

**Reply by bamboodigital-dev** [reply]
Just figured it out, are you using VS Code Insider? Do you have main copilot not nightly, and chat installed?

**Reply by pbrink231** [reply]
Also did all tese several times.  Including fully restarting the computer.  Tried again this morning after a couple of days.  Finally working.
- in VS Code - Insiders
- search "@enabled github" in extensions, for some reason said it needed a reload and clicked it. (did the reload when I installed it).
- make sure only gitub copilot nightly is installed (not chat or copilot)
- Still no luck, profile in bottom left -> myname (github) -> sign out
- vs code will reload on signout, prompt to sign in will popup, click on that, click ok on next 2 prompts.  
- Suddenly chat icon finally showed.

**Comment by For-Chance** [comment]
![image](https://github.com/community/community/assets/92280914/31a1ea17-938f-4bac-829e-3e63e759f424)
![image](https://github.com/community/community/assets/92280914/75fcbdb3-be03-4f5d-a975-78a3c9ca3d0e)
I had received the email, downloaded the Visual Studio Code Insider and Github Copilot Nightly. But Copilot Chat Icon disappears. What should I do?

**Reply by johndavis92790** [reply]
Same here, I have tried every reinstallation and other ideas on this page and I still can't get it to show up! Very disappointing. I was excited when I got the email, now very frustrated.

**Reply by johndavis92790** [reply]
It showed up once and then I moved it to the right side of my vscode to be separate from my main activity bar and then when I went to open another repo, it disappeared. Maybe there is a bug with separating it from the main activity bar? This happened on both my windows and mac.

**Reply by For-Chance** [reply]
I think you're right! I moved it to the right side, too. And next time I opened the window, the Chat Icon disappeared!

**Reply by For-Chance** [reply]
> It showed up once and then I moved it to the right side of my vscode to be separate from my main activity bar and then when I went to open another repo, it disappeared. Maybe there is a bug with separating it from the main activity bar? This happened on both my windows and mac.

oh，thank you very much! I think I have solved this problem! Only you need to do is open the Secondary Side Bar.
![image](https://github.com/community/community/assets/92280914/13779987-fc0c-45d1-be0a-22cd6f63fa83)

**Reply by KilinW** [reply]
> > It showed up once and then I moved it to the right side of my vscode to be separate from my main activity bar and then when I went to open another repo, it disappeared. Maybe there is a bug with separating it from the main activity bar? This happened on both my windows and mac.
> 
> oh，thank you very much! I think I have solved this problem! Only you need to do is open the Secondary Side Bar. ![image](https://user-images.githubusercontent.com/92280914/239200473-13779987-fc0c-45d1-be0a-22cd6f63fa83.png)

Thank you so much bro. I move it from activity icon to right side yesterday and I can't find it any more. It just hidden in the secondary sidebar lol.

**Comment by trentsteel84** [comment]
I've had an email saying i'm off the waitlist. So i installed it to VS 17.5.5.
On my personal computer it works fine, but on my work computer i get loads of popup errors on VS starting up, saying similar things to "The 'TestWindowPackage' package did not load correctly. The problem may have been caused by a configuration change..."
It stops all projects from being able to load. Restarting didn't help as the popup suggests. Uninstalling the extension makes everything work again. And reinstalling didn't make any difference.

**Reply by trentsteel84** [reply]
I've just updated to version 0.1.651.34310 and it's now working 👍

**Comment by soupCodez** [comment]
Hmm... it seems to work on Windows for me but not on Linux (EndeavourOS KDE).

UPDATE: The AUR package seems to install a version of VSCode Insiders that is incompatible with the Copilot chat extension. I will try again today and keep you updated.

**Reply by JamesPerrotta** [reply]
Any update? I'm on vanilla Arch (KDE), and I'm not seeing the chat icon.

**Reply by soupCodez** [reply]
> Any update? I'm on vanilla Arch (KDE), and I'm not seeing the chat icon.

I will keep you updated once I try again on Linux.

**Reply by JamesPerrotta** [reply]
No need. :)

Looks like it's working on my end. I forgot to update VSCode Insiders.

**Reply by soupCodez** [reply]
> No need. :)
> 
> Looks like it's working on my end. I forgot to update VSCode Insiders.

Glad to hear that. I will test again soon to see if it works on my end.

**Comment by nilshulth** [comment]
So I got it to work after a lot of trial and error. I am on a Mac. Here is what I did:
* I completely logged out from Github in VS Code Insiders and then quit the app
* I uninstalled both copilot-nightly and copilot-chat using: `code-insiders --uninstall-extension github.copilot-nightly` and `code-insiders --uninstall-extension github.copilot-chat` 
* I removed all residues of those extensions: `rm -rf ~/.vscode-insiders/extensions/github.copilot-*`
* I restarted VS Code Insiders
* I reinstalled Github Copilot Nightly from within VS Code Insiders
* I authenticated with Github from the accounts icon in the sidebar
Then it worked again!
Maybe not all the steps are needed, but that's what I did

**Reply by gamousquesORT** [reply]
Thanks, I just followed the first two steps and worked fine. I also chose the wrong profile.

**Reply by philippjbauer** [reply]
I followed you steps and it's still not working for me.

VS Code: 1.79.0-insider
Copilot Nightly: v1.86.92
Copilot Chat: v0.1.2023051601

**Reply by nilshulth** [reply]
Sorry to hear that. Strange problem. Maybe try deinstalling VS Code Insiders too?

**Comment by Dave-lab12** [comment]
I was having the same problem the issue was I was using regular vscode had to install vscode insiders then it worked

**Comment by orikahmadov** [comment]
I have tried uninstalling Github Copilot and installing the Github Copilot Nightly AND Github Copilot Chat. But it is not possible to install Github Copilot Chat because "it has no release version".

Guys dont bother yourself installing VSCode Insiders, you will be able to install Copilot Chat extension there but the chat icon wont still show up 

![chbchat](https://github.com/community/community/assets/55695784/1c8c84e8-299b-40eb-bbb4-6d0b9fddc35f)

**Reply by nilshulth** [reply]
But you are not supposed to install Copilot Chat, the instruction from GitHub says to install Copilot Nightly, and with that you will get the chat as a side effect.

**Reply by orikahmadov** [reply]
![ghbchat2](https://github.com/community/community/assets/55695784/66e061da-8548-4b0c-b499-5d1ec2720d8e)

Here it is mate, uninstalled Github Copilot and re-installed  the Nightly + Chat, as you see no chat icon displayed in the left column

**Reply by orikahmadov** [reply]
> But you are not supposed to install Copilot Chat, the instruction from GitHub says to install Copilot Nightly, and with that you will get the chat as a side effect.

Then only side effect you are getting is your nerves ahahah

**Reply by nilshulth** [reply]
Truly sorry to hear that, very annoying bug :-(

**Reply by Dave-lab12** [reply]
Install vscode insiders it doesn't work on regular vscode

**Reply by orikahmadov** [reply]
> Install vscode insiders it doesn't work on regular vscode

Hi Dave,
Here is another full screenshot, just for the sake of it I have installed the VSCode Insiders as you can see from the green icon below. It still does not work. Nobody from Github addresses this issue.
I think there might be 2 reasons for this
1.Capacity of Copilot Chat (Since its using ChatGPT, they might have limited capacity of computational power)
2.They have not approved my access to it. (Some people got access within hours after joining to waiting list) no idea how does it actually work.

I have been on waiting list for longer period than most of people above. The only thing I have not tried is, testing it on my Linux. But just from reading above comments, I can see that Mac, Linux owners are facing the same issue. 


![ghb_chat](https://github.com/community/community/assets/55695784/329ac67c-4401-4bc7-b186-65bb4e184c88)

**Reply by orikahmadov** [reply]
Currently I ve installed Copilot Nightly + Copilot Chat extension (latest relased versions for both extensions)

**Reply by davecheney** [reply]
@orikahmadov we have you on the waitlist, please be patient. You will receive an email when you've been accepted.

**Reply by davecheney** [reply]
@Dave-lab12 that is correct, chat only works in -insiders at the moment.

**Reply by orikahmadov** [reply]
> @orikahmadov we have you on the waitlist, please be patient. You will receive an email when you've been accepted.

Thaaaaaaanks so much for replying. 🙂 Can't wait to see it

**Reply by orikahmadov** [reply]
![image](https://github.com/community/community/assets/55695784/aa110d6b-17a4-421c-bfef-bee18664661f)


FINALLY I HAVE RECEIVED THE EMAIL FROM GITHUB!

**Reply by orikahmadov** [reply]
![fin_chat](https://github.com/community/community/assets/55695784/b03c1b98-3da1-414e-9adc-c06d72867ef1)


Then I started VSCode Insiders and the icon was there! Thank you Github

**Reply by jpike88** [reply]
@davecheney I'm waiting patiently now for some time... please accept me on copilot x waitlist!!! thanks!!!!!

**Comment by amalv** [comment]
I find it a little strange that when launched, Copilot X was advertised as including chat and you have 60 days to try it and afterwards you are charged, but I only have a few more days until the 60 days run out and I'm still on the waiting list:

![test](https://github.com/community/community/assets/1252707/2eba9f63-0b6b-4163-a06a-503d8a43cacb)

So, to recap, you have 60 days to try the product but you can't really try the product, or at least one of its main usages (the chat box).

Not sure how difficult it can be, but it would be really nice that Github would start the trial days count after you are accepted in the Copilot Chat waitlist.

**Comment by amalv** [comment]
Just an update, I've been accepted in the private beta, after that, what I had to do was to install VSC insider, the Github Copilot Chat extension and login using my github account, after that it worked like a charm :)

**Comment by ElhananGitHub** [comment]
I got an email that i was accepted, **install the VS Code insiders** and the Copilot Nightly extension, and its working perfect!
<img width="549" alt="Screen Shot 2023-05-23 at 13 54 17" src="https://github.com/community/community/assets/54417323/895c38e7-abb2-4000-8a30-1a06a6676fc2">

**Comment by bfeist** [comment]
OMG, this was the issue. It hides the chat icon when it's open, even on a sidebar not shown. https://github.com/microsoft/vscode-copilot-release/issues/102#issuecomment-1561011935

**Comment by kylewalow** [comment]
I had the same problem with the chat icon not showing up after following the guide sent via email.
For me, the solution was to explicitly trust the Github Copilot Nightly Build like so:
![image](https://github.com/community/community/assets/48148789/1d1cd2f0-8ef8-49d0-81a6-428aecf36139)

**Reply by stephenjstraley** [reply]
What is the solution for Visual Studio (not VS Code)... if you know?

**Comment by rcheungmv** [comment]
How long can we use Copilot Chat?

**Comment by ElijahLynn** [comment]
I'm on Linux and having trouble getting the chat to work:

* I'm using code-insiders SNAP on Ubuntu
* I'm using Copilot Nightly extension
* The Copilot Nightly extension auto-enables the Copilot chat extension
* I'm logged in with my GitHub account and I authorized GH through the web browser

No worky. One thing of note though is that the Copilot Chat extension says we need to accept the terms here https://docs.github.com/en/early-access/copilot/github-copilot-chat-technical-preview-license-terms but I don't see a way to accept them. 


edit: I've also installed Copilot Labs which has a similar interactive sidebar for accepting inputs, but not chat. This does work after I accepted the Next agreement that it prompted me to do. 

I feel like the agreement needs to be accepted here https://docs.github.com/en/early-access/copilot/github-copilot-chat-technical-preview-license-terms but don't know how to agree.

**Reply by ElijahLynn** [reply]
Working now on Linux!

Last night after I posted this (8:50pm PT), at 10:12pm PT, I got an email "You're now in the GitHub Copilot chat private beta!". I saw it this morning and went to VSCode and saw that one of the Copilot extensions wanted an update or did do an update and wanted a reload. I think I did reloaded and possibly signed in/out etc. And now it is working and is awesome!

It works just like Interviewing.io's AI Interviewer, which was my motivation for seeking out Copilot Chat! I'm gonna learn so much faster now!!!

**Comment by rocioDEV** [comment]
Anyone else with this error?

`Failed to install CopilotChat {"name":"Error","message":"Missing gallery"}`

**Reply by returnednull** [reply]
I'm here for this error also, but I'm still in waitlist.

**Reply by rocioDEV** [reply]
I reinstalled Copilot Chat extension and it is all ok now :) (I have access to the beta)

**Reply by stephenjstraley** [reply]
Are you using Visual Studio 2022?

**Comment by zvalcav** [comment]
I didn't receive any email. Yesterday I hit the Wait list link again and it told me I'm off the wait list already. Try that too if you are waiting for long time.

**Comment by zook111** [comment]
my chat icon suddenly disappeared, noticed connection  issue with github.appi in logs
this sorted it:
sudo sh -c "echo nameserver 8.8.8.8 > /etc/resolv.conf"

**Comment by roughgreys** [comment]
I encountered the same problem.After passing the application, I was able to use it normally on Mac OS, but on Windows, the sidebar did not display. My solution was to first install Github Copilot Nightly in Visual Code Insiders, then click on 'Disable Globally' and then 'Enable Globally' in the bottom right corner. An additional extension 'Copilot Chat' will then automatically appear in the extensions, and the sidebar will also appear.

**Comment by kross-aeg** [comment]
No luck getting Chat to work even though I received the "**You're now in the GitHub Copilot chat private beta!**" email.

- Windows 10. 
- Latest VS Code
- Installed Gihub Copilot (Nightly deprecated now). 
- Signed into GitHub in VS Code and allowed, etc.
- Enabled Settings Sync
- No Chat icon in sidebar.
- Sad Panda :(

**Comment by fqh4** [comment]
Microsoft Visual Studio Community 2022 (64-bit) - Current Version 17.6.5

This happens all the time:

![image](https://github.com/community/community/assets/125499340/55fda98a-1da3-4340-b2c4-cb5fd58d95c1)

![image](https://github.com/community/community/assets/125499340/63f8d67e-745c-4fbe-9e53-71e45cbac08a)

**Reply by commonsensedevs** [reply]
I use GitHub copilot chat for Business. I can use it in Visual Studio. I have Copilot chat extension installed but its not showing in my activity bar.

**Comment by albertord84** [comment]
Any help with Gihub Copilot (Nightly deprecated now)? There is another way to see Gihub Copilot Chat without Gihub Copilot Nightly?

It's not possible to find Gihub Copilot Nightly any more :(

**Comment by justinmchase** [comment]
When I look in the copilot chat output window I see these lines:

```
2023-09-27T18:55:25.980Z [INFO] [auth] Logged in as jchase24_uhg
2023-09-27T18:55:27.356Z [INFO] [chat] copilot token chat_enabled: false
2023-09-27T18:55:27.356Z [INFO] [auth] Got Copilot token for jchase24_uhg
```

I have access to copilot through my enterprise and org, do I need to do something additional to get access to copilot chat?

**Reply by wejy** [reply]
In enterprise Copilot settings check for Copilot Chat status. For my org it was disabled by default. Switching it to enabled made the Chat icon visibile.

**Reply by justinmchase** [reply]
So an Enterprise Admin would need to switch this on? The enterprise copilot settings in GitHub not VSCode?

**Reply by wejy** [reply]
Correct.

**Comment by wweevv-johndpope** [comment]
chat not working. sighh........
where are the logs - how can we give dig up logs that would be helpful and relevant??
just seems like a backend capacity problem as the  extension has 2.5 million installs. 
Ive installed on ubuntu 22.04 / mac / windows 11 - 
I've installed with vs code insiders - nothing. 
maybe I try vpn - this is BS.
https://github.com/microsoft/vscode-copilot-release/issues/429

@digitarald - please communicate that this is a problem - there's heap of tickets logged - and silence. 
It would be better to say - "this service is not available until further notice" - than just take my money and have me dig through support tickets wondering why this doesn't work. Is it geo-blocked?

There's pages and pages of these - going back to July? 
https://github.com/microsoft/vscode-copilot-release/issues?page=2&q=is%3Aissue+is%3Aopen+chat

**Comment by digitarald** [comment, marked as answer]
Thanks for your patience and feedback on this issue. We have consolidated all the troubleshooting steps for this problem in https://github.com/microsoft/vscode-copilot-release/issues/436. We have enabled more accounts and resolved some bugs since the first reports came in, so you should encounter this issue much less often now.

**Comment by rpijoan-lsw** [comment]
I'm in VSCode Insiders with GitHub Copilot Nightly enabled and it seems is still happening randomly.

My solution was to disable globally, reload the window and then re-enable the extension.

**Comment by mikkorantalainen** [comment]
If the Copilot Chat is still missing, you may be experiencing incorrectly set organization policy which results in silent failure:
https://github.com/microsoft/vscode-copilot-release/issues/551

**Reply by justinmchase** [reply]
There is an org setting but as it turns out your org (my org) may not have a license agreement that allows it to be enabled, so I'm told.

**Reply by Lebski** [reply]
It also disappeared for all beta testers yesterday (after it was unlocked for the organisations and disabled per default)

**Reply by jayarjo** [reply]
Does it have to be org? So doesn't work for individual accounts and repos?

**Comment by natemate90** [comment]
If your `Copilot Chat in the IDE` setting is disabled, it won't work. You can see the status here: [https://github.com/settings/copilot](https://github.com/settings/copilot)

**Reply by dvelle** [reply]
perfect this worked.

**Comment by plinio-s-borges** [comment]
I closed by mistake the chat window and it desapeared. I found the sollution by pressing Ctrl + P than typing Chat and selecting Chat: Open Session in Side Bar

![image](https://github.com/community/community/assets/49813172/2069799e-cd8f-43ca-8cca-2ff7168c176d)

**Reply by ikegentz** [reply]
Thank you, this happened to me as well! Seemed to close itself upon rebooting my machine

**Reply by AClon314** [reply]
I put it on the other activity bar, this helpes thx! `Chat: Open Chat in Side bar`

**Comment by gamesover** [comment]
I found download the latest version will show the chat now

**Comment by pogopo789** [comment]
![Screenshot from 2024-11-30 16-27-14](https://github.com/user-attachments/assets/e4b18159-2084-4e40-912c-68dedc98cecf)
Right click on the bar and enable "Chat"
